<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Shubham" failures="1" tests="9" name="Test" time="87.207" errors="0" timestamp="2021-11-03T18:39:50 IST">
  <testcase classname="TestClasses.MultipleDataLogin" name="dataUsername1Invalid" time="9.098">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestClasses.MultipleDataLogin.dataUsername1Invalid(MultipleDataLogin.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- dataUsername1Invalid -->
  <testcase classname="TestClasses.MultipleDataLogin" name="dataUsername2Valid" time="4.785"/>
  <testcase classname="TestClasses.MultipleDataLogin" name="dataUsername3Invalid" time="4.803"/>
  <testcase classname="TestClasses.MultipleDataLogin" name="dataUsername4Valid" time="4.56"/>
  <testcase classname="TestClasses.MultipleDataLogin" name="dataUsername5Invalid" time="4.567"/>
  <testcase classname="TestClasses.MultipleDataLogin" name="dataUsernamePassword1Invalid" time="7.409"/>
  <testcase classname="TestClasses.MultipleDataLogin" name="dataUsernamePassword2Valid" time="8.126"/>
  <testcase classname="TestClasses.MultipleDataLogin" name="dataUsernamePassword3Valid" time="7.216"/>
  <testcase classname="TestClasses.MultipleDataLogin" name="dataUsernamePassword4Invalid" time="5.293"/>
</testsuite> <!-- Test -->
